{
	"id": "CVE-2022-28346",
	"repositories": [
		{
			"allow_forking": true,
			"created_at": "2022-04-25T08:27:34Z",
			"description": "SQL injection in QuerySet.annotate(), aggregate(), and extra()",
			"forks": 2,
			"forks_count": 2,
			"full_name": "YouGina/CVE-2022-28346",
			"html_url": "https://github.com/YouGina/CVE-2022-28346",
			"id": 485285000,
			"name": "CVE-2022-28346",
			"owner": {
				"avatar_url": "https://avatars.githubusercontent.com/u/5002624?v=4",
				"html_url": "https://github.com/YouGina",
				"id": 5002624,
				"login": "YouGina"
			},
			"pushed_at": "2022-04-25T11:18:12Z",
			"stargazers_count": 1,
			"updated_at": "2023-07-18T07:41:07Z",
			"visibility": "public",
			"watchers": 1,
			"watchers_count": 1
		},
		{
			"allow_forking": true,
			"created_at": "2022-04-26T14:47:56Z",
			"description": "Django QuerySet.annotate(), aggregate(), extra() SQL 注入",
			"forks": 5,
			"forks_count": 5,
			"full_name": "DeEpinGh0st/CVE-2022-28346",
			"html_url": "https://github.com/DeEpinGh0st/CVE-2022-28346",
			"id": 485828921,
			"name": "CVE-2022-28346",
			"owner": {
				"avatar_url": "https://avatars.githubusercontent.com/u/34375573?v=4",
				"html_url": "https://github.com/DeEpinGh0st",
				"id": 34375573,
				"login": "DeEpinGh0st"
			},
			"pushed_at": "2022-05-31T03:28:34Z",
			"stargazers_count": 23,
			"updated_at": "2024-08-12T20:22:46Z",
			"visibility": "public",
			"watchers": 23,
			"watchers_count": 23
		},
		{
			"allow_forking": true,
			"created_at": "2022-05-15T00:24:19Z",
			"description": "An issue was discovered in Django 2.2 before 2.2.28, 3.2 before 3.2.13, and 4.0 before 4.0.4. QuerySet.annotate(), aggregate(), and extra() methods are subject to SQL injection in column aliases via a crafted dictionary (with dictionary expansion) as the passed **kwargs.",
			"full_name": "ahsentekd/CVE-2022-28346",
			"html_url": "https://github.com/ahsentekd/CVE-2022-28346",
			"id": 492352752,
			"name": "CVE-2022-28346",
			"owner": {
				"avatar_url": "https://avatars.githubusercontent.com/u/23294573?v=4",
				"html_url": "https://github.com/ahsentekd",
				"id": 23294573,
				"login": "ahsentekd"
			},
			"pushed_at": "2022-05-15T00:53:22Z",
			"updated_at": "2024-09-13T11:00:52Z",
			"visibility": "public"
		},
		{
			"allow_forking": true,
			"created_at": "2023-03-04T01:43:58Z",
			"description": "An issue was discovered in Django 2.2 before 2.2.28, 3.2 before 3.2.13, and 4.0 before 4.0.4. QuerySet.annotate(), aggregate(), and extra() methods are subject to SQL injection in column aliases via a crafted dictionary (with dictionary expansion) as the passed **kwargs.",
			"full_name": "vincentinttsh/CVE-2022-28346",
			"html_url": "https://github.com/vincentinttsh/CVE-2022-28346",
			"id": 609374925,
			"name": "CVE-2022-28346",
			"owner": {
				"avatar_url": "https://avatars.githubusercontent.com/u/14941597?v=4",
				"html_url": "https://github.com/vincentinttsh",
				"id": 14941597,
				"login": "vincentinttsh"
			},
			"pushed_at": "2022-05-15T00:53:22Z",
			"updated_at": "2022-08-24T08:06:59Z",
			"visibility": "public"
		},
		{
			"allow_forking": true,
			"created_at": "2024-01-19T00:15:59Z",
			"description": "A flaw was found in the Django package, which leads to a SQL injection. This flaw allows an attacker using a crafted dictionary containing malicious SQL queries to compromise the database completely.",
			"full_name": "kamal-marouane/CVE-2022-28346",
			"html_url": "https://github.com/kamal-marouane/CVE-2022-28346",
			"id": 745257468,
			"name": "CVE-2022-28346",
			"owner": {
				"avatar_url": "https://avatars.githubusercontent.com/u/110986028?v=4",
				"html_url": "https://github.com/kamal-marouane",
				"id": 110986028,
				"login": "kamal-marouane"
			},
			"pushed_at": "2024-01-19T04:00:05Z",
			"stargazers_count": 1,
			"updated_at": "2024-01-20T05:30:50Z",
			"visibility": "public",
			"watchers": 1,
			"watchers_count": 1
		}
	]
}
